lib/order.rb

```diff
@@ -0,0 +1,43 @@
+require_relative 'message'
+require_relative 'menu'
+
+class Order
+
+attr_reader :basket, :menu, :sms
+
+  def initialize(menu = Menu.new)
+    @menu = menu.items
+    @basket = {}
+  end
+
+  def view_menu
+    @menu.list
+  end
+
+  def add_to_order(name, quantity = 1)
+    raise 'Item is unavailable' unless @menu.include?(name)
+    @basket[name] = quantity
+  end
+
+  def order_total
+    @basket.map do |name, quantity|
+      @menu[name] * quantity
+    end
+  end
+
+  def sum_order
```

Comment:
Good - Design for Single Responsibility Principle ✅
I like that your methods are small and easy to read, it makes the code much better